interface/orders/types.php

```diff
@@ -31,276 +29,377 @@
+                <form method='post' name='theform'
+                    action='types.php?popup=<?php echo $popup ?>&order=<?php
+                    echo $order;
+                    if (isset($_GET['formid'])) {
+                        echo '&formid=' . $_GET['formid'];
+                    }
+
+                    if (isset($_GET['formseq'])) {
+                        echo '&formseq=' . $_GET['formseq'];
+                    }
+                    ?>'>
+                    <div class="btn-group">
+                        <button type="button" name="form_search" class="btn btn-default btn-refresh" onclick="refreshme()"><?php echo xlt('Refresh');?></button>
+                        <button type="button" class="btn btn-default btn-add" name='add_node_btn' id='add_node_button'  onclick='anode(0)'><?php echo xlt('Add Top Level');?></button>
+                    </div>
+                    <br>
+                    <br>
+                    <div class = "table-responsive">
+                        <table class="table" style="margin-bottom:0">
+                            <thead>
+                                <tr class='head'>
+                                    <td class='col1' align='left'>&nbsp;&nbsp;<?php echo xlt('Name') ?> <i id="name-tooltip" class="fa fa-info-circle oe-text-black" aria-hidden="true"></i></td>
+                                    <td class='col2 oe-pl0' align='left'><?php echo xlt('Category') ?> <i id="order-tooltip" class="fa fa-info-circle oe-text-black" aria-hidden="true"></i></td>
+                                    <td class='col3 oe-pl0' align='left'><?php echo xlt('Code') ?> <i id="code-tooltip" class="fa fa-info-circle oe-text-black" aria-hidden="true"></i></td>
+                                    <td class='col6 oe-pl0' align='left'><?php echo xlt('Tier') ?> <i id="tier-tooltip" class="fa fa-info-circle oe-text-black" aria-hidden="true"></i></td>
+                                    <td class='col4 oe-pl0' align='left'><?php echo xlt('Description') ?></td>
+                                    <td class='col5 oe-pl0' align='left' style='padding-left:0px'><?php echo xlt('Edit/Add') ?></td>
+                                </tr>
+                            </thead>
+                        </table>
+                    </div>
+                    <div id="con0"></div>
+                    <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
+                    <div class="form-group clearfix">
+                        <div class="col-sm-12 text-left position-override">
+                            <div class="btn-group btn-group-pinch" role="group">
+                                <?php if ($popup) { ?>
+                                    <button type="submit" class="btn btn-default btn-save" name='form_save' value='<?php echo xlt('Save'); ?>'><?php echo xlt('Save');?></button>
```

Comment:
the first xlt should be an xla on line above

Reply:
sorry will fix

Reply:
Done